assets/stylesheets/components/_crds-components-skeleton-blocks.scss

```diff
@@ -21,6 +21,13 @@
 
 .shared-header-skeleton {
   background-color: $cr-cyan;
+
+  svg {
```

Comment:
Let's use a class for this so we don't run into issues if an app decides to throw in another SVG for some reason. Maybe `shared-header-skeleton-logo` just to be super specific?